README.md

```diff
@@ -10,214 +10,208 @@
+```js
+import pw from 'playwright-core';
 
-You still execute the script itself which gives you total control over what library you want to choose and when to do upgrades. This also comes with the benefit of keep your code proprietary and able to run on numerous platforms. We simply take care of all the browser-aspects and offer a management layer on top of the browser.
+const browser = await pw.firefox.connect(
+  'ws://localhost:3000/firefox/playwright'
+);
 
-### Docker
+const page = await browser.newPage();
+await page.goto('https://example.com');
+console.log(await page.title());
+await browser.close();
+```
 
-> [!TIP]
-> See more options on our [full documentation site](https://docs.browserless.io/baas/docker/quickstart).
+ **Note:** Use `ghcr.io/browserless/firefox` or `ghcr.io/browserless/multi` for Firefox/Webkit support.
 
-1. `docker run -p 3000:3000 ghcr.io/browserless/chromium`
-2. Visit `http://localhost:3000/docs` to see the documentation site.
-3. See more at our [docker package](https://github.com/browserless/browserless/pkgs/container/base).
+</details>
 
-### Hosting Providers
+</br>
 
-We offer a first-class hosted product located [here](https://browserless.io). Alternatively you can host this image on just about any major platform that offers hosting for docker. Our hosted service takes care of all the machine provisioning, notifications, dashboards and monitoring plus more:
+**Output:**
+```
+Example Domain
+```
 
-- Easily upgrade and toggle between versions at the press of a button. No managing repositories and other code artifacts.
-- Never need to update or pull anything from docker. There's literally zero software to install to get started.
-- Scale your consumption up or down with different plans. We support up to thousands of concurrent sessions at a given time.
 
-If you're interested in using this image for commercial aspects, then please read the below section on licensing.
+## ✨ Features
 
-### Puppeteer
+### General Features
 
-Puppeteer allows you to specify a remote location for chrome via the `browserWSEndpoint` option. Setting this for Browserless is a single line of code change.
+- **Parallelism and queueing** — Handle multiple sessions with configurable concurrency limits
+- **Debug Viewer** — Actively view and debug running browser sessions in real-time
+- **Unforked libraries** — Works seamlessly with standard Puppeteer and Playwright
+- **Fonts & emoji** — All system fonts and emoji support out-of-the-box
+- **Configurable timeouts** — Set session timers and health-checks to keep things running smoothly
+- **Error tolerant** — If Chrome crashes, Browserless won't
+- **Apple M series support** — Full support for development and running on ARM architecture
 
-**Before**
+### Premium Features
```

Comment:
there is alot more we can add here like persisting sessions, chrome extensions, session replay, captcha/stealth routes